public/index.html

```diff
@@ -101,6 +101,5 @@
 
 <div id="content"></div>
 <script src="https://code.jquery.com/jquery-3.1.0.js"></script>
-<script type="text/jsx" src="index.js"></script>
```

Comment:
@jasonrclark that is done by webpack through the npm react-scripts build step. All the files are built and staged in the build directory. The gh-pages deploy script copies that directory up to github, but that is all.